### ISSUE TITLE
[0.49.9] Personnel Market paid recruitment rolls option does nothing

### ISSUE THREAD

**Issue by kipstafoo** [MegaMek/mekhq#1940]
Selecting the 'paid recruitment roll' in the personnel market doesn't do anything. The type of personnel didn't show up for 2 months and I had no 100k finance deduction for at all (expected to have 8 of them over 2 months).

**Comment by Windchild292** [comment]
Do you have an active contract and are you playing AtB?

**Comment by Windchild292** [comment]
Also, I cannot reproduce this on my test campaign and will require a campaign file to do so.

**Comment by Kendricke** [comment]
Curious if this is related to https://github.com/MegaMek/mekhq/issues/1952 

I have a campaign file in that one if it helps (it's a similar issue where I stopped getting ANY personnel market updates during a contract).  I just confirmed that with that campaign, if I manually completed the Mission and began attempting paid recruitment rolls, I would get nothing. 

I would get "hangs" around Mondays each week, but no actual personnel market changes.
